fastai/callback/fp16.py

```diff
@@ -71,22 +79,34 @@ def get_master(opt, flat_master=False):
     return model_params, master_params
 
 # Cell
-def to_master_grads(model_pgs, master_pgs, flat_master=False):
+def to_master_grads( model_pgs:[[nn.Parameter]], # Fp16 model parameters to copy gradients from
+    master_pgs:[[nn.Parameter]], # Fp32 model parameters to copt gradients to
+    flat_master:bool=False, # Whether or not fp32 parameters were previously flattened
+):
+    "Move fp16 model gradients to fp32 master gradients"
     for (model_params,master_params) in zip(model_pgs,master_pgs):
         model_grads_to_master_grads(model_params, master_params, flat_master=flat_master)
 
 # Cell
-def to_model_params(model_pgs, master_pgs, flat_master=False)->None:
+def to_model_params(model_pgs:[[nn.Parameter]], # Fp16 model params to copy to
+    master_pgs:[[nn.Parameter]], # Fp32 master params to copy from
+    flat_master:bool=False # Whether master_pgs was previously flattened
+)->None:
+    "Copy updated fp32 master params to fp16 model params after gradient step. "
     for (model_params,master_params) in zip(model_pgs,master_pgs):
         master_params_to_model_params(model_params, master_params, flat_master=flat_master)
 
 # Cell
-def test_overflow(x):
+def test_overflow(x:torch.Tensor # Fp16 gradients
+):
+    "Tests whether fp16 gradients have overflowed."
     s = float(x.float().sum())
     return (s == float('inf') or s == float('-inf') or s != s)
 
 # Cell
-def grad_overflow(pgs):
+def grad_overflow(pgs:[[nn.Parameter]] # Fp16 parameter groups to test for overflow
+)->bool:
+    "Tests all fp16 parameters in pgs for gradient overflow"
```

Comment:
I am under the impression that for these one argument methods we can expand the doc string if needed in leu of a docment
```suggestion
def grad_overflow(pgs:[[nn.Parameter]])->bool:
    "Tests all fp16 parameters in pgs for gradient overflow"
```